bootstrap.js

```diff
@@ -14,9 +14,9 @@ function startup(data, reason) {
 function shutdown(data, reason) {
   console.log('bootstrap shutdown called');
   // no teardown is needed for a normal shutdown
-  if (reason == APP_SHUTDOWN) { return; }
+  if (reason === APP_SHUTDOWN) { return; }
 
-  if (reason == ADDON_DISABLE || reason == ADDON_UNINSTALL) {
+  if (reason === ADDON_DISABLE || reason === ADDON_UNINSTALL) {
```

Comment:
I have no idea what I'm doing here... Plus, ESLint complains these are undefined. If we cant use `===` here, I can work on silencing the ESLint warnings for these lines/blocks.